README.md

```diff
@@ -448,6 +448,40 @@ DispatchRider.config do |config|
   config.error_handler = DispatchRider::AirbrakeErrorHandler
 end
 ```
+## Scheduling Job
+
+Scheduled jobs currently require `ActiveRecord`. Support for other ORM will be added in the future depending on demand.
+
+1. Add and run the DB migration below:
+  ```ruby
+  class CreateScheduledJobsTable < ActiveRecord::Migration
+    include DispatchRider::ScheduledJob::Migration
+
+    def change
+      create_scheduled_jobs_table
+    end
+  end
+  ```
+
+2. Run publishing.
+  ```ruby
+  # Run once
+
+  # Ideally run on a cron. Where the cron is responsible for the publishing
+  # frequency. Any jobs due at the time this is run by the cron  will be
+  # published.
+  DispatchRider::ScheduledJob.publish_due_jobs
+
+  # Loop. Warning: Loops are blocking. Run this on a separate thread if it's not
+  # the sole purpose of the app.
+
+  # publish every minute
+  DispatchRider::ScheduledJob.publish_due_jobs every: 1.minute
+
+  # publish every half an hour
+  DispatchRider::ScheduledJob.publish_due_jobs every: 30.minutes
+  ```
+
```

Comment:
Here's the PR for the ActiveJob support: https://github.com/payrollhero/activejob-dispatch_rider/pull/5